### DISCUSSION TITLE
Removing the current page summary

### DISCUSSION THREAD

**Discussion by marco-silvestri** [filamentphp/filament#13032] [Help]
### Package

Table builder

### Package Version

v3.0

### How can we help you?

Hi! I am using a Sum::class summarizer. Is there any possibility to remove the summary for the current page and just leave the total?

**Comment by leandrocfe** [comment]
https://github.com/filamentphp/filament/blob/3.x/packages/tables/resources/views/components/summary/index.blade.php

The row is displayed because `$hasPageSummary` is true.

1 - You can create a PR.
2 - Alternatively, you can publish this file and edit it as needed, although we do not recommend this approach: https://filamentphp.com/docs/3.x/support/style-customization#publishing-blade-views

**Reply by marco-silvestri** [reply]
Thanks, I'll look into it!

**Comment by oliverbj** [comment]
Has there been any changes to this - or do we need to publish the views still?

**Reply by howdu** [reply]
Doesn't look like it but you can do it with custom css to save publishing the views
```css
tr.fi-ta-summary-header-row + tr.fi-ta-summary-row {
    display: none;
}
```

**Reply by marcelaodev** [reply]
> Doesn't look like it but you can do it with custom css to save publishing the views
> 
> ```css
> tr.fi-ta-summary-header-row + tr.fi-ta-summary-row {
>     display: none;
> }
> ```

It does work but kinda annoying since it will still run the query for the page.

This feature would be nice. I don't really care about page summaries...

**Comment by mc-fu** [comment]
Any news to this? Running into the same requirement. CSS is ok, but an option to disable this completly would be really great

**Reply by alisalehi1380** [reply]
👍

**Comment by alisalehi1380** [comment]
```css
table.fi-ta-table tbody tr.fi-ta-summary-row:nth-last-child(2){
    display: none;
}
```
